lib/index.js

```diff
@@ -0,0 +1,382 @@
+var _ = require('lodash'),
+    sdk = require('postman-collection'),
+
+    FIRST_LINE = /(.*?)\n.*/g,
+
+    requestOrResponse = function (what) {
+        return sdk.Request.isRequest(what) && 'request' || sdk.Response.isResponse(what) && 'response' || undefined;
+    },
+
+    statusClassReasons = {
+        info: 1,
+        success: 2,
+        redirection: 3,
+        clientError: 4,
+        serverError: 5
+    },
+
+    statusCodeReasons = {
+        // ok: 200, // cannot add 'ok' directly since it violates underlying 'ok' property
+        accepted: 202,
+        withoutContent: 204, // @todo add 1223 status code
+        badRequest: 400,
+        unauthorised: 401,
+        unauthorized: 401,
+        forbidden: 403,
+        notFound: 404,
+        notAcceptable: 406,
+        rateLimited: 429
+    };
+
+/**
+ * Adds postman-collection assertions via the chai-postman plugin interface.
+ *
+ * @param {Object} chai - A Chai assertion wrapper class provided by the chai library.
+ */
+module.exports = function (chai) {
```

Comment:
Put this inside another closure and accept SDK instance. 

```
module.exports = function (sdk) {
  return function (chai) {
    // …
  };
}
```

Reply:
Addressed.